pkg/pillar/containerd/oci.go

```diff
@@ -217,3 +221,29 @@ func (s *ociSpec) updateFromImageConfig(config v1.ImageConfig) error {
 	}
 	return oci.WithAdditionalGIDs("root")(ctrdCtx, CtrdClient, &dummy, &s.Spec)
 }
+
+// UpdateMounts
+func (s *ociSpec) UpdateMounts(disks []types.DiskConfig) {
+	for i, disk := range disks {
+		// Skipping root container disk
+		if i == 0 {
+			continue
+		}
+		mount := specs.Mount{}
+		access := ""
+		if disk.ReadOnly {
+			access = "rbind:ro"
+		} else {
+			access = "rbind:rw"
+		}
+		mount.Type = "bind"
+		mount.Source = path.Join("/var", disk.FileLocation)
```

Comment:
We should really cleanup this /var/persist vs. /persist business, but for now, can you please at least create a translation function? I believe there may be a few other place in oci/containerd implementation that would benefit from this translation -- fixing those would be bonus points ;-)